multihost_test/Bugzillas/test_bz_automation.py

```diff
@@ -355,3 +365,202 @@ def test_bz_951743(self, multihost):
         assert execute_cmd(multihost, "grep '^bz951743:\$' /etc/shadow").returncode == 0
         assert execute_cmd(multihost, "grep '^bz951743' /etc/shadow").returncode == 0
         execute_cmd(multihost, "userdel -rf bz951743")
+
+    @pytest.mark.tier1
+    def test_bz690829_1(self, multihost):
+        """
+        :title: Checks if useradd is able to copy files
+            from /etc/skel also if /home not mounted with acl option.
+        :id: 5d372fe0-4c88-11ee-9152-845cf3eff344
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=690829
+        :steps:
+          1. we are expecting /home exists
+          2. create files/dirs under skel
+          3. create new user
+          4. check if setfacl doesn't work
+          5. check if all files copied and correctly
```

Comment:
s/check if all files copied and correctly/check if all files are copied correctly

Reply:
This comment applies to all test cases